support/Pipelines/Bbh/InitialData.py

```diff
@@ -216,8 +273,47 @@ def generate_id(
     help=(
         "Radial expansion velocity adot0 which is radial velocity over radius."
     ),
-    required=True,
 )
+@click.option(
+    "--eccentricity",
+    "-e",
+    type=float,
+    help=(
+        "Eccentricity of the orbit. Specify together with _one_ of the other"
+        " orbital parameters. Currently only an eccentricity of 0 is supported"
+        " (circular orbit)."
+    ),
+)
+@click.option(
+    "--mean-anomaly-fraction",
+    "-l",
+    type=float,
+    help=(
+        "Mean anomaly of the orbit divided by 2 pi, so it is a number between 0"
+        " and 1. The value 0 corresponds to the pericenter of the orbit"
+        " (closest approach), the value 0.5 corresponds to the apocenter of the"
+        " orbit (farthest distance), and the value 1 corresponds to the"
+        " pericenter again."
+    ),
+)
+@click.option(
+    "--num-orbits",
+    type=float,
+    help=(
+        "Number of orbits until merger. Specify together with a zero"
+        " eccentricity to compute initial orbital parameters for a circular"
+        " orbit."
+    ),
+)
+@click.option(
+    "--time-to-merger",
+    type=float,
+    help=(
+        "Time to merger. Specify together with a zero eccentricity to compute"
+        " initial orbital parameters for a circular orbit."
+    ),
+)
```

Comment:
While adding these, I'm not sure how much of an effect tidal deformability has for including matter terms. @moble @ffoucart do you know what SpEC does for NSNS initial parameters? Just assume BHs?

Reply:
The main difference would be in the estimated number of orbits to merger. The current initial data just uses the BBH formula at a fixed distance. We do have an estimate of the number of orbits that includes some correction from the BBH formula, but I don't recall whether it's used at all in current automation. @tiamartineau would know.

Reply:
Alright, thank you! I'm just curious for the future, I don't think this should be worried about in this PR.

Reply:
No, not used just yet-- though it is set up in the input files and is ready to be used. Just haven't gotten to the actual ecc reduction automation to use it yet. (That is, if you're talking about the following in DMR.input?:)
```
my $Omega = $ID_Omega0;
my $OmegaMeanMotion  =  $Omega;


my $EccRedFinalTime = 1e9;  # i.e. effectively no trigger if not EccRedRun
if ($EccRedRun) {
    my $tmin = 600;  # Start time of ecc red fitting regime
    $EccRedFinalTime = $tmin + 5*pi/$OmegaMeanMotion;  # Stop after 2.5 orbits
}
```

Reply:
...ah, ignore my first comment, thought you were talking about the orbits to eccentricity reduction, not merger-- the answer is no, not yet.

Reply:
Okay, that's @tiamartineau !!